projects/js-packages/social-previews/composer.json

```
@@ -0,0 +1,48 @@
+{
+	"name": "automattic/social-previews",
+	"description": "A suite of components to generate previews for a post for both social and search engines.",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"automattic/jetpack-changelogger": "@dev"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"build-development": [
+			"pnpm run build"
+		],
+		"build-production": [
+			"NODE_ENV=production pnpm run build"
+		],
+		"test-coverage": [
+			"pnpm run test-coverage"
+		],
+		"test-js": [
+			"pnpm run test"
+		]
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true,
+	"extra": {
+		"autotagger": true,
```

Probably want to auto-publish to npm too:
```suggestion
		"autorelease": true,
		"autotagger": true,
		"npmjs-autopublish": true,
```
For the autopublish to work, I think you may also need to do https://docs.npmjs.com/trusted-publishers#step-1-add-a-trusted-publisher-on-npmjscom on the npm side. Possibly only after the mirror repo gets its first push.